miner/worker.go

```diff
@@ -1111,6 +1111,15 @@ func (w *worker) commitRip7560TransactionsBundle(env *environment, txs *types.Ex
 		env.gasPool = new(core.GasPool).AddGas(gasLimit)
 	}
 
+	// adding RIP-7560 bundle header transaction here
+	bundleHeaderTx := types.NewTx(&types.Rip7560BundleHeaderTx{
+		ChainID: w.chainConfig.ChainID,
+		// TODO: if some transactions are invalid, do we need to drop the bundle? or fix TransactionCount at that time?
+		TransactionCount: 1, //uint64(len(txs.Transactions)),
+		BlockNumber:      env.header.Number,
+		TransactionIndex: 0,
+	})
```

Comment:
Maybe crafting this `bundleHeaderTx` after validating and executing all txs, and then append it ahead of `validatedTxs` can work, right?

Reply:
Yes, that's a good idea. I've modified the structure to reflect your feedback.